### ISSUE TITLE
Notifications stopped after years of being fine

### ISSUE THREAD

**Issue by wildvortex** [taxilian/OctoPrint-Twilio#26]
I don't know when it started as I had not printed anything for a coupel weeks but noticed yesterday that I am no longer getting notifications. I checked my Twillio account balance and have plenty of money in it. I even deleted my plugin data on octoprint and put everything in again and still am not getting notifications.

**Comment by tedder** [comment]
They transitioned a lot of services to Kore, and also require 10DLC for texting. Perhaps you missed one or both of those?

**Comment by wildvortex** [comment]
I am not a programmer. This is part of an addon for "octoprint" so maybe the addon designer has missed this.

**Comment by taxilian** [comment]
@wildvortex neither of the things that @tedder mentioned are programming things -- both have to do with your Twillio account.

It's worth taking some time and use another tool that uses your Twillio account and see if you are able to send text messages using anything else -- then report back.

**Comment by wildvortex** [comment]
Yes, looks like I missed the 10DLC for texting. I am not sure if my registering has fixed the issue yet. Only use Twillio for the printer so not really on there much. Had some time to dig around and saw the stuff about the registration. never noticed it while looking on my phone. I found that there is a usage log and has several outgoing messages that show undelivered and that it is due to the 10DLC.

**Comment by wildvortex** [comment]
Thank you both, will register and test when done. likely will solve the issue.

**Comment by smeisner** [comment]
@wildvortex I am going thru the same process. I got my outgoing all set until I had to agree to paying another $2 / month for the "campaign" ... are you doing this as well?

**Comment by wildvortex** [comment]
> @wildvortex I am going thru the same process. I got my outgoing all set until I had to agree to paying another $2 / month for the "campaign" ... are you doing this as well?

Yes, I Paid the both of them. Not sure it is totally worth it but I would doubt that Twillo would refund the money already on my account. So stuck doing it. I don't know how long the regitration takes for approval but I do know that I have registered and tested and have not gotten the texts yet. I will give it a day then start contacting Twillo to get it sorted.

**Comment by wildvortex** [comment]
> @wildvortex I am going thru the same process. I got my outgoing all set until I had to agree to paying another $2 / month for the "campaign" ... are you doing this as well?

Yah, in my haste I did not see the $2/mo. Then I wake this morning and find that they changed me for just 1 number 3x the amount that they had stated. Tomorrow when I get the time I am contacting them and trying to get my money back and releasing the number. Tell them what they can do with their services

**Comment by wildvortex** [comment]
Ok, what either I did not see or was not on the actual registration page is that not only do they charge $2/mo for the "campaign" but also a one time $15 vetting fee.
Will be finding another way to receive notifications!!!

**Comment by smeisner** [comment]
I am currently investigating Amazon Web Services SNS feature (google search "aws sns"). It does cost $2 per month for the toll free number, but that's all ... I think. Still researching it.

**Comment by wildvortex** [comment]
Yah, I am thinking that I want to find something possibly free. Printer is WiFi based so maybe something where it sends a message to Google chat or something.

I reopened this in hopes that you would see it Smeisner

**Comment by wildvortex** [comment]
> I am currently investigating Amazon Web Services SNS feature (google search "aws sns"). It does cost $2 per month for the toll free number, but that's all ... I think. Still researching it.

If that doesn't work take a look at octoeverywhere. I think their email notification is free and if you want an SMS notification then you can become a supporter and have a bunch of different perks for under $50/year. 

I am hoping their email notification is free and have signed up to receive them. Won't know if they are free until 15 or so days from now as they have given me a free trial. Had already been using them to monitor prints while away but this might help. I just really want something to tell me when I am out in my shop in the garage if something goes wrong or the print is finished. There are all kinds of things you can set to get emails about.

**Comment by wildvortex** [comment]
Just remembered (by an email from them) free app and free push notifications.
![Screenshot_20230911_121715_Chrome](https://github.com/taxilian/OctoPrint-Twilio/assets/33760190/dcd48581-c2cc-4cde-97bb-eadf100ae961)
![Screenshot_20230911_121704_Chrome](https://github.com/taxilian/OctoPrint-Twilio/assets/33760190/8a4a76ab-f1ae-433a-87a8-e76e8ae64286)
